Plan/common/src/main/java/com/djrapitops/plan/delivery/webserver/ResponseFactory.java

```diff
@@ -254,6 +254,17 @@ public Response faviconResponse() {
         }
     }
 
+    public Response robotsResponse() {
+        try {
+            return Response.builder()
+                    .setMimeType(MimeType.FAVICON)
```

Comment:
This seems like a potential issue, I think robots.txt [should be served with text/plain mimetype](https://stackoverflow.com/questions/20819569/robots-txt-error-content-type-should-be-text-plain)
```suggestion
                    .setMimeType("text/plain")
```